test/test_skip_systemd.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative "helper"
+require_relative "helpers/integration"
+
+require "puma/plugin"
+
+class TestSkipSystemd < TestIntegration
+
+  THREAD_LOG = TRUFFLE ? "{ 0/16 threads, 16 available, 0 backlog }" :
+                 "{ 0/5 threads, 5 available, 0 backlog }"
```

Comment:
Hmm, do we need this here? :)

I see we added it in https://github.com/puma/puma/pull/3079 that this is probably based on, but maybe it wasn't needed there too (feel free to remove it from there now as well if it isn't needed)

Reply:
Done!